docs/BR.md

```diff
@@ -1651,9 +1650,7 @@ b. `cRLDistributionPoints`
 
 c. `authorityInformationAccess`
 
-   With the exception of stapling, which is noted below, this extension MUST be present. It MUST NOT be marked critical, and it MUST contain the HTTP URL of the Issuing CA's OCSP responder (accessMethod = 1.3.6.1.5.5.7.48.1). It SHOULD also contain the HTTP URL of the Issuing CA's certificate (accessMethod = 1.3.6.1.5.5.7.48.2).
-
-   The HTTP URL of the Issuing CA's OCSP responder MAY be omitted provided that the Subscriber "staples" OCSP responses for the Certificate in its TLS handshakes [RFC4366].
+   This extension MUST be present. It MUST NOT be marked critical, and it MUST contain the HTTP URL of the Issuing CA's OCSP responder (`accessMethod` = 1.3.6.1.5.5.7.48.1). It SHOULD also contain the HTTP URL of the Issuing CA's certificate (`accessMethod` = 1.3.6.1.5.5.7.48.2).
```

Comment:
Same comment as above. If Microsoft insists on requiring the AIA OCSP responder for all cases without exceptions, it should be clear.

Reply:
@dzacharo This is a very unhelpful comment. I can understand that you may feel frustrated, but this is an existing requirement of the Microsoft Root Program, which is clear there, and is clear here.

If you have a suggestion you'd like to make it clearer, I'm certainly open, but I don't think I can extract anything actionable from this comment that is useful or productive :(

Reply:
Sure, you are the author and I am trying to review a rather large ballot. Forgive me for not always having comments you find useful. As you mentioned in [a previous comment](https://github.com/sleevi/cabforum-docs/pull/10#discussion_r440333217), Microsoft requires OCSP for end-entity certificates bur requires OCSP or CRLDP for "other cases". I don't know why the Browsers decided to follow Microsoft's Policy instead of -say- Mozilla's which seems to address the OCSP responder hammering issue I mentioned and is aligned with the existing BRs.

I have no strong feelings about this but thought it should be raised so we have a clear path, based on existing requirements and no unintentional consequences.